src/PowerShellEditorServices/Services/TextDocument/Handlers/CodeLensHandlers.cs

```diff
@@ -43,11 +42,7 @@ public CodeLensHandlers(ILoggerFactory factory, SymbolsService symbolsService, W
             _logger = factory.CreateLogger<FoldingRangeHandler>();
             _workspaceService = workspaceService;
             _symbolsService = symbolsService;
-            _providers = new ICodeLensProvider[]
-            {
-                new ReferencesCodeLensProvider(_workspaceService, _symbolsService),
-                new PesterCodeLensProvider()
-            };
+            _codeLensProviders = _symbolsService.GetCodeLensProviders();
```

Comment:
is there an instance where a CodeLens provider would be added later and then this would be out of date?

Reply:
Good thought